etc/kayobe/kolla/globals.yml

```diff
@@ -112,3 +113,8 @@ barbican_api_image: "{% raw %}kolla/{{ kolla_base_distro }}-{{ barbican_install_
 barbican_keystone_listener_image: "{% raw %}kolla/{{ kolla_base_distro }}-{{ barbican_install_type }}-barbican-keystone-listener{% endraw %}"
 barbican_worker_image: "{% raw %}kolla/{{ kolla_base_distro }}-{{ barbican_install_type }}-barbican-worker{% endraw %}"
 
+# Use source build for blazar images as packages are not available
+blazar_install_type: "source"
+
+# Use source build for nova images so that blazar-nova is included
+nova_install_type: "source"
```

Comment:
Is `nova_install_type: source` necessary? Looks like the placement api container is missing a binary.

Reply:
Fixed by https://review.opendev.org/#/c/696876/